insights/parsers/virsh_list_all.py

```diff
@@ -79,14 +79,17 @@ def parse_content(self, content):
         if not content:
             return
 
-        self.cols = parse_fixed_table(content,
-                                      heading_ignore=['Id', 'Name', 'State'],
-                                      header_substitute=[('Id', 'id'), ('Name', 'name'), ('State', 'state')])[1:]  # noqa
-        self._cleanup()
-
-        for item in self.cols:
-            self.fields.append(self.keyvalue(item['name'], item['state'], item['id'], item['name'].lower()))  # noqa
-        self.keywords = [name.name_lower for name in self.fields]
+        try:
+            self.cols = parse_fixed_table(content,
+                                          heading_ignore=['Id', 'Name', 'State'],
+                                          header_substitute=[('Id', 'id'), ('Name', 'name'), ('State', 'state')])[1:]  # noqa
+            self._cleanup()
+
+            for item in self.cols:
+                self.fields.append(self.keyvalue(item['name'], item['state'], item['id'], item['name'].lower()))  # noqa
+            self.keywords = [name.name_lower for name in self.fields]
+        except ValueError:
+            self.keywords = []
```

Comment:
To give some context about this issue, the issue(ValueError) occurs when the parser don't see the three headers: **Id, Name and State** in the collected output and throws the error message:

```shell
Line containing 'Id,Name,State' was not found in table 
```

Even if the output has no real data, this parser does not complains. For example:

```shell
 Id    Name                           State
----------------------------------------------------
```

I've seen(from hadoop analysis on insights archives) this happen only when the `virsh` binary is not available and the data collected in the file virsh_list_all is: 

```shell
timeout: failed to run command '/usr/bin/virsh': No such file or directory 
```

Reply:
@psachin - As per the new [Components and Exceptions](https://insights-core.readthedocs.io/en/latest/exception_model.html) policy of insights core, all `Empty` and `Unexpected` content which breaks the parser, should be skipped or captured.

Fore this parser `VirshListAll`, since it's a sub-class of [CommandParser](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/__init__.py#L509), the `No such file or directory` can be Skipped by raising a `ContentException`. So, the same to this `ValueError: Line containing 'Id,Name,State' was not found in table`, a ParseException should be raised instead of returning an empty Object.

Reply:
Done

Reply:
@bfahr - I have discussed this with @psachin about which one is better: to raise `SkipException` or `ParseException` for the `ValueError`. This `ValueError` to me is a kind of issue that we should capture and do more search on it to analyze if there are any unexcepted format/content in the collected output, then to improve the logic of the parser.

If it's necessary to capture it, and since it's already captured by this [line-275](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/__init__.py#L275) in the original code, seems we do not need to apply this patch committed by this PR.

How do you think?